README.md

```diff
@@ -0,0 +1,34 @@
+
+Welcome to the JDK!
+===================
+
+For information about building the JDK, including how to retrieve all
+of the source code, please see either of these files:
+
+  * doc/building.html   (html version)
+  * doc/building.md     (markdown version)
+
+See http://openjdk.java.net/ for more information about the OpenJDK
+Community and the JDK.
+
+---
+About Project Panama
+===================
+This is a repo of OpenJDK Project Panama: Interconnecting JVM and native code. You can find more information at: https://openjdk.java.net/projects/panama/
```

Comment:
This repository contains changes which aims at improving the interoperability between the Java programming language and native libraries, which one of the main goals of Project Panama. More information can be found at: https://openjdk.java.net/projects/panama/